### PR TITLE
Data fixes

### PR DESCRIPTION
**Pull request by ricburton**

### CODE REVIEW

File: Balance/Table Cells/CryptoBalanceTableViewCell.swift [hunk 1]
```diff
@@ -287,7 +288,21 @@ private class CryptoRow: UIView {
             make.top.equalTo(iconImageView).offset(-2)
         }
 
-        tokenNameLabel.text = name
+        var tokenName = "No Name"
+
+        if let name = name {
```
Comment (borwahs) [inline Balance/Table Cells/CryptoBalanceTableViewCell.swift:293]: Just FYI, if we wanted to let iOS handle it, we can set a property on the UILabel to add an ellipsis. This solution works too if we want to hardcode a certain number of chars.

Comment (borwahs) [reply Balance/Table Cells/CryptoBalanceTableViewCell.swift:293]: https://developer.apple.com/documentation/uikit/nslinebreakmode/bytruncatingtail

Comment (ricburton) [reply Balance/Table Cells/CryptoBalanceTableViewCell.swift:293]: Oh sweet. I'll merge this for now as a few people have complained about the 0 tokens. I am not super familiar with changing parts of a pull request.

Comment (einsteinx2) [reply Balance/Table Cells/CryptoBalanceTableViewCell.swift:293]: @ricburton in the future, to change stuff in your open PRs, just make new commits to the branch and push them

**Review by einsteinx2 — APPROVED** [review]

**Pull request merged**